lib/server.go

```diff
@@ -54,7 +60,7 @@ func (s *HTTPServer) handleInfoSwitch(res http.ResponseWriter, req *http.Request
 }
 
 func (s *HTTPServer) handleCmd(res http.ResponseWriter, req *http.Request) {
-
+  enableCors(&res)
```

Comment:
Because we changed the function definition to not accept a reference to an interface, no need for the ampersand, here: `enableCors(res)`.